app/assets/stylesheets/administrate/components/_container.scss

```diff
@@ -0,0 +1,6 @@
+.container {
+  background-color: $white;
+  box-shadow: 0 2px 4px rgba($grey-7, 0.24), 0 2px 6px rgba($grey-7, 0.12);
```

Comment:
It definitely might be useful in the future, but for now, since it’s only used in this one place, I think keeping it static is fine. Once we see repetition we can variablize…